src/content/docs/en/pages/secure-journey/advanced-configurations/automate-origin-shield.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Automate Origin Shield
+description: Create a rule to automate the use of the Origin Shield list and block potential harmful IPs.
+meta_tags: edge, secure, origin shield, list, rules engine, behavior
+namespace: docs_secure_automate_origin_shield
+menu_namespace: secureMenu
+permalink: /documentation/guides/secure/automate-origin-shield/
+---
+
+import Button from '~/components/Button.astro'
+
+**Origin Shield** provides a list of IP addresses used in Azion's Edge.
```

Comment:
```suggestion
**Origin Shield** provides a list of IP addresses used in Azion's Edge Network.
```

Nodes? Platform?

Reply:
ou **Origin Shield** provides a list of Azion Edge Nodes' IP addresses.